Colors for different battery levels

haoxiangliew opened the pull request:
Current levels are set to 10% (Red) 50% (Yellow) and 100% (Green)

josteink commented on src/workspacer.Bar/Widgets/BatteryWidget.cs:
```diff
@@ -11,8 +11,12 @@ namespace workspacer.Bar.Widgets
         public bool HasBatteryWarning { get; set; } = true;
         public double LowChargeThreshold { get; set; } = 0.10;
+        public double MedChargeThreshold { get; set; } = 0.50;
+        public double HighChargeThreshold { get; set; } = 1.00;
```
This value isn't used for anything and should be removed. 

Apart from that, this all looks good to me 👍

haoxiangliew replied:
Changed it to only have a Low Battery Color

josteink commented on src/workspacer.Bar/Widgets/BatteryWidget.cs:
```diff
@@ -28,23 +32,26 @@ public override IBarWidgetPart[] GetParts()
+                {
+                    return Parts(Part(currentBatteryCharge.ToString("#0%"), MedChargeColor));
+                }
+                else if (currentBatteryCharge <= HighChargeThreshold)
```
Same comment as last time: we don't need high charge threshold, and this else-is block should be merged with the following else-block.

josteink commented:
I think maybe you removed too much? This is the entire PR now: :smile: 

![image](https://user-images.githubusercontent.com/411338/93565571-deaef600-f98b-11ea-85f9-f447345bbde7.png)

haoxiangliew commented:
Just changed the file, and removed the redundant else function. Already in stable is the low battery color (red), my commit only adds two levels of color to the battery widget for medium battery (less than 50%) and high battery (above 50%)

haoxiangliew commented:
Also the new layouts are in another PR now :D

josteink commented:
Looks much better!

Before we merge, could you please remove the "HighChargeThreshold" variable, since it is no longer in use :)

haoxiangliew commented:
Done!

josteink commented:
LGTM!

Thanks for the patch!

haoxiangliew commented:
No problem. thanks for reviewing my PR!

rickbutton commented:
thanks for the PR @haoxiangliew and thanks for reviewing @josteink

josteink commented:
Np, Rick. 

As for the layouts PR, I'll leave that to someone else. Don't think that's my area of expertise 😉
